cmd/agent/main.go

```diff
@@ -111,30 +97,29 @@ func newService(ctx context.Context, logger mglog.Logger, eventSvc events.Servic
 	return svc
 }
 
-// extractComputationValue to extract computation value from the command line.
-func extractComputationValue() (agent.Computation, error) {
-	cmdLineBytes, err := os.ReadFile("/proc/cmdline")
+func readConfig() (agent.Computation, error) {
+	l, err := vsock.Listen(manager.ManagerPort, nil)
```

Comment:
Why `l`? This is confusing. Why not `vs` if this is vsocket?

Reply:
`l` for listener

Reply:
Yep, all clear - thanks.